docs/tutorials/getting-started-with-node-js.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Getting Started with Node JS
+group: software-languages
+author: "@liam-b"
+---
+
+{% include /style/icon.html type="warning" %}
+Note that Node JS run really slowly on the EV3 brick and is limited to NPM version 0.10.29 which can be a problem when trying to install some libraries.
```

Comment:
It's also limited to an equivalently old Node.js version, which should also be mentioned.